crates/bevy_ecs/src/world/mod.rs

```diff
@@ -788,6 +788,12 @@ impl World {
         unsafe { self.get_resource_with_id(component_id) }
     }
 
+    pub fn get_many_res<'w, MR: ManyRes<'w>>(&'w self) -> MR::Item {
+        MR::get_resources(self)
+    }
+    pub fn get_many_res_mut<'w, MR: ManyResMut<'w>>(&'w mut self) -> MR::Item {
```

Comment:
```suggestion
    pub fn get_many_resources_mut<'w, MR: ManyResMut<'w>>(&'w mut self) -> MR::Item {
```

Reply:
> If we're only going to have one pair of this API, I think I would prefer a panicking version of this akin to `world.resource`: these are generally more useful. Unwrapping all of the resources here is pretty tedious.

If this is the behavior we go with, should we also remove `get_` from the naming?

Reply:
> If this is the behavior we go with, should we also remove get_ from the naming?

Exactly :)

> or I suppose just have all four methods like the non-tuple versions?

Yep, this is the alternative, but I really don't think that the `get` versions will be useful enough to warrant cluttering the API. They can also be manually reconstructed if really needed.